src/i-shared-minter-mapping.ts

```diff
@@ -0,0 +1,163 @@
+    event.params._projectId
+  );
+  if (!project) {
+    log.warning("Project {} not found for core contract {}", [
+      event.params._projectId.toString(),
+      event.params._coreContract.toHexString()
+    ]);
+    return;
+  }
+
+  // load minter
+  const minter = loadOrCreateMinter(event.address, event.block.timestamp);
+
+  // load or create project minter configuration
+  const projectMinterConfig = loadOrCreateProjectMinterConfiguration(
+    project,
+    minter
+  );
+
+  projectMinterConfig.currencyAddress = event.params._currencyAddress;
+  projectMinterConfig.currencySymbol = event.params._currencySymbol;
+  projectMinterConfig.save();
+
+  // update project's updatedAt timestamp to induce a sync
+  // @dev this may induce a sync if the project's minter is not this minter,
+  // but good to consider the project as "updated", as it could switch to this
+  // minter in the future
+  project.updatedAt = event.block.timestamp;
+  project.save();
+}
+
+export function handleProjectMaxInvocationsLimitUpdated(
+  event: ProjectMaxInvocationsLimitUpdated
+): void {
+  // attempt to load project, if it doesn't exist, log a warning and return
+  // @dev we don't support or allow minters to pre-configure projects that do
+  // not yet exist
+  const project = tryLoadProject(
+    event.params._coreContract,
+    event.params._projectId
+  );
+  if (!project) {
+    log.warning("Project {} not found for core contract {}", [
+      event.params._projectId.toString(),
+      event.params._coreContract.toHexString()
+    ]);
+    return;
+  }
+
+  // load minter
+  const minter = loadOrCreateMinter(event.address, event.block.timestamp);
+
+  // load or create project minter configuration
+  const projectMinterConfig = loadOrCreateProjectMinterConfiguration(
+    project,
+    minter
+  );
+
+  projectMinterConfig.maxInvocations = event.params._maxInvocations;
+  projectMinterConfig.save();
+
+  // update project's updatedAt timestamp to induce a sync
+  // @dev this may induce a sync if the project's minter is not this minter,
+  // but good to consider the project as "updated", as it could switch to this
+  // minter in the future
+  project.updatedAt = event.block.timestamp;
+  project.save();
+}
+
+///////////////////////////////////////////////////////////////////////////////
+// EVENT HANDLERS end here
+///////////////////////////////////////////////////////////////////////////////
+
+///////////////////////////////////////////////////////////////////////////////
+// HELPER FUNCTIONS start here
+///////////////////////////////////////////////////////////////////////////////
+/**
+ * Helper function to attempt to load a project from the store. If the project
+ * does not exist, returns null.
+ * @param coreContractAddress core contract address of the project
+ * @param projectNumber project number of the project (BigInt)
+ * @returns The Project entity from the store if it exists, otherwise null
+ */
+function tryLoadProject(
```

Comment:
Nit: `tryLoadProject` feels a bit misleading as a name here because of the association with `try` being something that would catch an error that we could handle. I think just `loadProjectByCoreAddressAndProjectNumber` or something would be fine since the behavior of returning null is inline with the normal entity load function.

Reply:
agreed, updated to suggested name in [67668ca](https://github.com/ArtBlocks/artblocks-subgraph/pull/229/commits/67668ca36c262bc1fc42b341b054d42fbb0d86f7)